src/bundle/lib.css

```diff
@@ -0,0 +1,374 @@
+@tailwind base;
```

Comment:
Maybe we should add a small comment here as well, to explain the reason for this file?